hibernate-core/src/main/java/org/hibernate/persister/collection/AbstractCollectionPersister.java

```diff
@@ -718,25 +718,22 @@ protected void logStaticSQL() {
 
 		MODEL_MUTATION_LOGGER.debugf( "Static SQL for collection: %s", getRole() );
 
-		if ( getRowMutationOperations().hasInsertRow() ) {
-			final String insertRowSql = getRowMutationOperations().getInsertRowOperation().getSqlString();
-			if ( insertRowSql != null ) {
-				MODEL_MUTATION_LOGGER.debugf( " Row insert: %s", insertRowSql );
-			}
+		final String insertRowSql = getRowMutationOperations().getInsertRowOperation() != null ?
+				getRowMutationOperations().getInsertRowOperation().getSqlString() : null;
```

Comment:
I guess our usual code style is to hit this with Introduce Variable:

```suggestion
		final JdbcMutationOperation insertRowOperation = getRowMutationOperations().getInsertRowOperation();
		final String insertRowSql = insertRowOperation != null ? insertRowOperation.getSqlString() : null;
```